docs/DeveloperGuide.adoc

```diff
@@ -714,6 +714,20 @@ We now have everything set up... but we still can't modify the remarks. Let's fi
 See this https://github.com/se-edu/addressbook-level4/pull/599[PR] for the step-by-step solution.
 
 [appendix]
+== Target Users
+
+This application is targeted towards University students, aiming to meet their needs throughout their University life, and hopefully beyond. University students meet a lot of people in the university, such as project group members, tutors, lecturers, friends and other connections. Nowadays it is important for students to build connections with other people, as connections may also help students for their future career after graduation. Since the students meet a lot of people, they may not be able to remember all the information about the people they met. Therefore, iContacts aims to help students to store and manage their contacts easily.
```

Comment:
"...aiming to meet their needs throughout their university life..." (change 'University' to lower case 'university')

"University students meet a lot of people in the university, such as project group members, tutors, lecturers, friends and other connections."
Students meet **many** people in the university, such as tutors, lecturers, friends, peers, project group members and other connections.